### PR TITLE
add support for postgres database url

### PR DESCRIPTION
**Pull request by markburns**

### CODE REVIEW

**Comment by jerser** [comment]
Wouldn't it be better to change the `database_config` method to support DATABASE_URL for all types?
Also, URI.parse could then be used? (e.g. https://gist.github.com/jakeonrails/2767211)

File: lib/alchemy/tasks/helpers.rb [hunk 1]
```diff
@@ -48,19 +48,31 @@ def mysql_command(cmd = 'mysql')
 
       def postgres_command(cmd = 'psql')
         command = []
-        if database_config['password']
-          command << "PGPASSWORD='#{database_config['password']}'"
+
+        config = database_config
```
Comment (fernandezvieira) [inline lib/alchemy/tasks/helpers.rb]: Not sure why you need this? Could just use `database_config` where `config` is being used?

Comment (fernandezvieira) [reply lib/alchemy/tasks/helpers.rb]: Ah, never mind. I misread it.

**Comment by markburns** [comment]
@jerser - kind of? It feels like it would be less clear about the reason why this code is added. We want this change because of alchemy, so putting the change there feels better.

Although I am facepalming that I didn't use URI.parse here.

**Review by fernandezvieira — APPROVED** [review]
LGTM

**Comment by jerser** [comment]
@markburns Yeah but don't we want this change to live in upstream Alchemy?

If we do it should change database_config so it works in both cases.
If we don't I don't see why it should be in our fork of alchemy and not as a monkey patch in author.

**Comment by markburns** [comment]
@jerser - we were using a fork anyway. not sure if we want it upstream or not, but this should make it more upstream-able, thanks for the suggestion. 

How's this now?

**Comment by jerser** [comment]
Yeah fine by me 👍 
Wasn't aware we already pointed to the fork in our Gemfiles, just feels better to be able to pull in upstream and not having to merge back in to our fork everytime.

**Comment by markburns** [comment]
@jerser - absolutely agree. And it's great if they are happy to merge straight off the bat, I'm just not a big fan of chasing up PR maintenance either.

**Comment by markburns** [comment]
Merging because tests actually passed on CI, we just need circle.yml setup properly (or move to travis to match upstream)

**Pull request merged**